cli/client/errors.go

```diff
@@ -20,3 +21,7 @@ type ServerError models.ServerError
 func (this *ServerError) Error() string {
```

Comment:
`this` doesn't seem to be consistent with style guidelines.
https://github.com/golang/go/wiki/CodeReviewComments#receiver-names